Mathlib/CategoryTheory/Limits/Yoneda.lean

```diff
@@ -222,4 +222,4 @@ end Functor
 
 end CategoryTheory
 
-assert_not_exists AddCommMonoid
+-- assert_not_exists AddCommMonoid
```

Comment:
Keeping this gave the following error:

```
Declaration AddCommMonoid is not allowed to be imported by this file.
It is defined in Mathlib.Algebra.Group.Defs,
  which is imported by Mathlib.Algebra.Group.Basic,
  which is imported by Mathlib.Algebra.Group.Embedding,
  which is imported by Mathlib.Data.Finset.Image,
  which is imported by Mathlib.Data.Fintype.Basic,
  which is imported by Mathlib.Data.Fintype.Card,
  which is imported by Mathlib.CategoryTheory.FinCategory.AsType,
  which is imported by Mathlib.CategoryTheory.Limits.Shapes.FiniteLimits,
  which is imported by Mathlib.CategoryTheory.Limits.Shapes.FiniteProducts,
  which is imported by Mathlib.CategoryTheory.Limits.FunctorCategory,
  which is imported by this file.

These invariants are maintained by `assert_not_exists` statements, and exist in order to ensure that "complicated" parts of the library are not accidentally introduced as dependencies of "simple" parts of the library.
```

I have no idea what its purpose was, but the error is clearly caused by changing imports

Reply:
I could revert the import change in `Limits.FunctorCategory`, and create a new file with the declarations about finite limits in functor categories?

Reply:
Having read the docstring of `assert_not_exists`, I think I definitely should revert this import change and create a new file instead.

Reply:
Indeed, creating a new file it the best solution.